apps/predbat/predbat.py

```diff
@@ -896,7 +896,7 @@
         "has_charge_enable_time": False,
         "has_discharge_enable_time": False,
         "has_target_soc": False,
-        "has_reserve_soc": True,
+        "has_reserve_soc": False,
```

Comment:
There is a reserve setting in the template configuration for GS, does it not work?